sawmill-core/src/main/java/io/logz/sawmill/processors/ExternalMappingSourceProcessor.java

```diff
@@ -197,4 +200,16 @@ public ExternalMappingSourceProcessor create(Map<String, Object> config) {
         }
     }
 
+    public static final class Constants {
+        private Constants() {}
```

Comment:
why do we need this constructor?

Reply:
to restrict creating objects of this class. It's not intended to be used as an instance, only as Constants holder.